webpack.server.config.js

```diff
@@ -0,0 +1,56 @@
+const webpack = require('webpack');
+const path = require('path');
+
+const ENV = process.env.NODE_ENV || 'development';
+
+const sConfig = {
+  name: 'server',
+  target: 'node',
+  entry: './server/server.js',
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    libraryTarget: 'commonjs2',
```

Comment:
We have to change the target, since we are targeting a commonjs environment, I think this should work with other configs in node, but this is the one that worked for me